charts/nginx-gateway-fabric/values.yaml

```diff
@@ -112,6 +115,8 @@ nginx:
     #     batchCount: 4
     #   serviceName: ""
     #   spanAttributes: []
+    # logging:
```

Comment:
Once #2494 is merged you will have to update the schema here

Reply:
#2492 is probably the right tag, I think I did this correctly and ran `make generate-helm-schema` and I think the generated json is correct.